client/src/components/Backlog.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const Backlog = ({ tasks }) => {
+  return (
+    <table className="task-table">
+      <thead className="task-header">
+        <tr>
+          <th>Titel</th>
+          <th>Beschrijving</th>
+          <th>Deadline</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+      <tbody className="task-body">
+        {tasks.map((task) => (
+          <tr key={task.id} className="task-row">
+            <td>{task.Title}</td>
```

Comment:
You have some properties (`tasks`, `id` ...) which are lower-cased and some which are capitalized (`Title`, `Description`, `Deadline` ...). If you find some time to fix this, I would put them all in lower (camel) cased, that's usually the convention for Javascript property keys.